src/app_service/service/contacts/service.nim

```diff
@@ -326,8 +372,7 @@ QtObject:
 
   proc removeContact*(self: Service, publicKey: string) =
     var contact = self.getContactById(publicKey)
-    contact.added = false
```

Comment:
I would leave `added = false`, since the base gaol is to remove them from our added contacts

Reply:
@jrainville the way of handling contacts is different now, if we leave this here, there is no chance to create appropriate condition and have "removed contact" added to the "rejected contacts model" cause `removeContact` proc is called if we want to reject one's contact request. Anyhow... all this is temporary here, since we need to revise all that once we add more flags on the `status-go` side to match new design.